nodeJS/express/installing_postgresql.md

```diff
@@ -123,16 +123,16 @@ You should see the PostgreSQL prompt come up with the new role we just created,
 role_name=#
 ```
 
-If you don't see a similar prompt, then reach out in [our Discord server](https://discord.gg/V75WSQG) for some help. If you **do** see a similar prompt, then we can create a password for the role like so:
+If you don't see a similar prompt, then reach out in [our Discord server](https://discord.gg/V75WSQG) for some help. If you **do** see a similar prompt, then we can create a password for the role like so (with a backward slash \):
 
 ```sql
 \password <role_name>
 ```
 
-You'll be prompted to enter a password and to verify it. Once you are done, the prompt will return to normal. Now, we will configure the permissions for our new role:
+You'll be prompted to enter a password and to verify it. Once you are done, the prompt will return to normal. Now, we will configure the permissions for our new role(Note the semicolon at the end):
```

Comment:
```suggestion
You'll be prompted to enter a password and to verify it. Once you are done, the prompt will return to normal. Now, we will configure the permissions for our new role (note the semicolon at the end):
```

Same applies for all instances of this addition.

Reply:
Still require a space between `role` and `(note`. The latest diff current shows you have `rote(note` instead of `role (note`